generators/server/templates/src/main/resources/swagger/api.yml.ejs

```diff
@@ -35,7 +35,46 @@ components:
       content:
         "application/problem+json":
           schema:
-            $ref: 'https://opensource.zalando.com/problem/schema.yaml#/Problem'
+            Problem:
```

Comment:
This line needs to be removed and review indentation

Reply:
Hello, do you mean remove the name of the object (Problem) and de-indent everything by one level?

Reply:
Yes. But apply first the 2 changes requested, and if it fails try this, please

Reply:
Should I do it in new commits or amend this?

Reply:
Like you want 🙂